packages/mongodb-memory-server-core/src/util/__tests__/dryBinary.test.ts

```diff
@@ -155,6 +157,46 @@ describe('DryBinary', () => {
     });
   });
 
+  describe('getBinaryName', () => {
```

Comment:
maybe also add a test for a different architecture and different platform combination (eg arm, windows)